dicom_standard/preprocess_modules_with_attributes.py

```diff
@@ -30,7 +38,8 @@ def preprocess_attribute_fields(tables):
 
 
 def preprocess_single_table(table):
-    table['attributes'] = list(map(preprocess_attribute, table['attributes']))
+    # Used in conjunction with lines 55-56 to catch exception
```

Comment:
I think it's a good idea to add a comment in explaining why this key was added, but probably should avoid using explicit line numbers in the comment. These line numbers could easily change and the comment will become stale. Perhaps provide a URL to the table in question that has the issue, or a link to the github issue that motivated the change?

Reply:
I'll update the comment. This was not due to an issue, but rather a change to the parsing code that makes it pull all module tables from section 3, not just ones in Chapter C.